### PR TITLE
Branch add logging

### PR DESCRIPTION
**Pull request by Cp-John**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/UndoCommand.java [hunk 1]
```diff
@@ -52,6 +54,8 @@ public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
         if (commandHistory.empty()) {
+            Logger julLogger = LogsCenter.getLogger(UndoCommand.class);
+            julLogger.info(MESSAGE_NO_COMMAND_TO_UNDO);
```
Comment (w2vgd) [inline src/main/java/seedu/address/logic/commands/UndoCommand.java]: What does `julLogger` stand for? Will it be better to use a more descriptive name?

**Review by w2vgd — COMMENTED** [review]
A suggestion to rename the variable to provide more clarity

File: src/main/java/seedu/address/logic/commands/UndoCommand.java (hunk 1 above)
Comment (Cp-John) [reply src/main/java/seedu/address/logic/commands/UndoCommand.java]: Thanks for your kind suggestion.

**Review by w2vgd — APPROVED** [review]
LGTM

**Pull request merged**